source/core/TypeAndDataManager.cpp

```diff
@@ -956,7 +996,20 @@ Boolean TypeCommon::IsComplex()
     }
     return false;
 }
-
+//------------------------------------------------------------
+Boolean TypeCommon::IsAnalogWaveform()
+{
+    TypeRef t = this;
+    while (t) {
+        SubString typeName = t->Name();
+        SubString analogTypess(TypeAnalogWaveform);
+        if (typeName.FindFirstMatch(&analogTypess, 0, false) == 0) {
```

Comment:
Please elaborate why are we doing a FindFirstMatch and not a Compare as in the TypeCommon::IsPath case. If we decide to have a type AnalogWaweformInternal in the future, it might find this one instead.